js/main.js

```diff
@@ -189,3 +189,29 @@ document.querySelectorAll('.see-project').forEach((btn) => {
     });
   });
 });
+
+// validation
+const form = document.querySelector('#contact form');
+const { email } = form.elements;
+
+function showMsg(input, msg) {
+  const small = document.createElement('small');
+  small.classList.add('error');
+  small.textContent = msg;
+  form.insertBefore(small, input);
+}
```

Comment:
- [ ] Good job so far :+1:  , however the show error message should be near the submit button as required, kindly fix it and request for another review